core/src/include/baconfig.h

```diff
@@ -85,34 +85,24 @@ void InitWinAPIWrapper();
 #ifdef ENABLE_NLS
 #  include <libintl.h>
 #  include <locale.h>
-#  ifndef _
-#    define _(s) gettext((s))
-#  endif /* _ */
+#  ifndef T_
+#    define T_(s) gettext((s))
+#  endif /* T_ */
 #  ifndef N_
 #    define N_(s) (s)
 #  endif /* N_ */
 #else    /* !ENABLE_NLS */
-#  undef _
+#  undef T_
 #  undef N_
 #  undef textdomain
 #  undef bindtextdomain
 #  undef setlocale
 
-#  ifndef _
-#    define _(s) (s)
-#  endif
-#  ifndef N_
-#    define N_(s) (s)
-#  endif
-#  ifndef textdomain
-#    define textdomain(d)
-#  endif
-#  ifndef bindtextdomain
-#    define bindtextdomain(p, d)
-#  endif
-#  ifndef setlocale
-#    define setlocale(p, d)
-#  endif
+#  define T_(s) (s)
+#  define N_(s) (s)
```

Comment:
What is the difference between `N_()` and `NT_()` ? Should we unify those to just `NT_()` ?